templates/cluster_resources.j2

```diff
@@ -1,6 +1,7 @@
 {%- set data = pillar.cluster.configure.template.parameters %}
 {%- set cloud_provider = grains['cloud_provider'] %}
 {%- set sid = data.sid.upper() %}
+{%- set sid_lower = sid.lower() %}
```

Comment:
should it not be `data.sid.lower()`

Reply:
It's the the same, check the `sid` is initialized just one line before.

Reply:
Yes i skipped that, thanks